src/shared/firewall-util-nft.c

```diff
@@ -924,6 +930,29 @@ static int fw_nftables_add_masquerade_internal(
         return sd_nfnl_call_batch(nfnl, &m, 1, NFNL_DEFAULT_TIMEOUT_USECS, NULL);
 }
 
+static int af_to_nfproto(int af) {
+        assert(IN_SET(af, AF_INET, AF_INET6));
+
+        switch(af) {
+        case AF_INET:
+                return NFPROTO_IPV4;
+        case AF_INET6:
+                return NFPROTO_IPV6;
+        }
+
+        return -1;
```

Comment:
please stick to our usual logic of returning negative error numbers. i.e. return -EINVAL or -EAFNOSUPPORT or so here

Reply:
ok